src/deploy/functions/runtimes/node/versioning.ts

```diff
@@ -16,6 +16,7 @@ interface NpmShowResult {
 }
 
 const MIN_SDK_VERSION = "2.0.0";
+const NPM_COMMAND_TIMEOUT_MILLIES = 3000;
```

Comment:
We discussed this internally and think that `10000` would be a more appropriate timeout for customers with slower connectivity or an intermittent issue

Reply:
@colerogers ok good point. I updated the value as per your recommendation